Wire-iOS/Sources/Helpers/UIBarButtonItem+StyleKit.swift

```diff
@@ -45,4 +45,57 @@ extension UIBarButtonItem {
         return item
     }
 
+    static func createHeaderDoneItem(
```

Comment:
I would rename that to

```suggestion
    static func createNavigationBarButtonDoneItem(
```

Reply:
Done.